package.json

```diff
@@ -3,7 +3,7 @@
         "@eslint/js": "^9.10.0",
         "@types/eslint__js": "^8.42.3",
         "@types/eslint-config-prettier": "^6.11.3",
-        "eslint": "^9.10.0",
+        "eslint": "9.14.0",
```

Comment:
This should probably go in another PR ahead of all others.

Reply:
If all ready to merge it, lets merge, otherwise ill open seperatly. but i actually hope you didn't waited for me to fix the CI with one char removal.